masterlist.yaml

```diff
@@ -14609,6 +14618,57 @@ plugins:
         crc: 0x7E36F0E5
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 122
+        
+  # Additional Clockwork
+  - name: 'ClockworkCourierCounterspell.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+    group: *fixesGroup
+    clean:
+      - crc: 0x6ABC5154
+        util: 'SSEEdit v4.0.3'
+  - name: 'ClockworkShadowmarksCastleExt.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+    msg:
+      - <<: *alreadyInX
+        subs: [ '"Additional Clockwork - Marked Shadows Merged" (ClockworkShadowmarksMerged.esp)' ]
+        condition: 'active("ClockworkShadowmarksMerged.esp")'
+    clean:
+      - crc: 0xC73F27D3
+        util: 'SSEEdit v4.0.3'
+  - name: 'ClockworkShadowmarksTermini.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+    msg:
+      - <<: *alreadyInX
+        subs: [ '"Additional Clockwork - Marked Shadows Merged" (ClockworkShadowmarksMerged.esp)' ]
+        condition: 'active("ClockworkShadowmarksMerged.esp")'
+    clean:
+      - crc: 0x93BA7A0D
+        util: 'SSEEdit v4.0.3'
+  - name: 'ClockworkShadowmarksTravelRoom.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+    msg:
+      - <<: *alreadyInX
+        subs: [ '"Additional Clockwork - Marked Shadows Merged" (ClockworkShadowmarksMerged.esp)' ]
+        condition: 'active("ClockworkShadowmarksMerged.esp")'
+    clean:
+      - crc: 0xEDC64E4B
+        util: 'SSEEdit v4.0.3'
+  - name: 'ClockworkShadowmarksMerged.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+    clean:
+      - crc: 0xB4C1E773
+        util: 'SSEEdit v4.0.3'
+  - name: 'ClockworkTitleRemover.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+    clean:
+      - crc: 0x913191E9
+        util: 'SSEEdit v4.0.3'
+  - name: 'ClockworkMausoleumDummyDoorFix.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+    group: *fixesGroup
```

Comment:
There's a good chance this will cause a sorting error for some load orders (With `JKs Skyrim.esp` for example).
Since `Clockwork.esp` is a master file for `ClockworkMausoleumDummyDoorFix.esp`.
```yaml
    group: *fixesGroup
```

Best to just remove it.
```suggestion
```